components/home/Dashboard.tsx

```diff
@@ -83,44 +62,143 @@ export default function Dashboard() {
       </View>
     </View>
   );
+};
+interface Transaction {
```

Comment:
We have a transaction type already, will be easier to use this instead of redefining 👍